lib/src/test/java/com/auth0/jwt/interfaces/VerificationTest.java

```diff
@@ -0,0 +1,162 @@
+package com.auth0.jwt.interfaces;
+
+import com.auth0.jwt.JWTVerifier;
+import org.junit.Test;
+
+import java.time.Instant;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasEntry;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.junit.Assert.assertThrows;
+
+/**
+ * Tests for any default method implementations in the {@link Verification} interface.
```

Comment:
I found that we have a default method that we have previously added in the Verification interface, since the point of default methods in the interface is to avoid breaking changes, and we are releasing a major that can have breaking changes, why not remove the default method and implement it the right way? 

This will break one method implementation if the user is implementing the Verification interface but I guess it should be fine. Your thoughts on this @jimmyjames

Reply:
Removing the default methods that exist in v3 makes sense. And while we could add a new method to the interface without a default implementation in a new major, if there's a reasonable default implementation to make upgrading versions easier, I prefer that. We can remove the default method implementations for existing default methods in a separate PR.